math/wide_integer/uintwide_t.h

```diff
@@ -1,3 +1,6 @@
+// NOLINTBEGIN(*)
```

Comment:
@ckormanyos very sorry but this line was added in error. It's generated by a script to ensure that violations of the checks listed in CNL's .clang-tidy don't fail in the copy of _uintwide_t.h_ that is copied into the _include/cnl_ directory. It shouldn't have been copied back into _ckormanyos/wide-integer_. Now, Clang-Tidy will ignore the most important file in the project! 

I've opened #165 to revert this change.

Reply:
> his line was added in error

Thank you @johnmcfarlane and good catch.

I had noticed that and already eliminated these changes, which are cycling in their entirety in #160